packages/app/src/Sidebar.tsx

```diff
@@ -38,6 +38,7 @@ function Sidebar(props: Props) {
           role="tab"
           aria-selected={tab === Tab.Explore}
           onClick={() => setTab(Tab.Explore)}
+          aria-label="Open explorer panel"
```

Comment:
Added some aria-labels to be able to query the element

Reply:
Very minor again, but taking example on VS Code, I would shorten those to `Explorer` and `Search`: 

![image](https://user-images.githubusercontent.com/2936402/228180906-15e26aa2-6a51-41cb-96c2-eb4dec07bf50.png)

Screen reader users would already know they're on a tab and understand that each tab opens the corresponding panel.